trezor-crypto/src/bip32.c

```diff
@@ -48,6 +48,15 @@
 #include <TrezorCrypto/sha3.h>
 #include <TrezorCrypto/sodium/keypair.h>
 
+const curve_info ed25519_hd_info = {
```

Comment:
This is identical to `ed25519_info`, why duplication is needed?

Reply:
We needed to create a new one so we can take a different path in the code. However, the curve info is exactly the same to the original ed25519 one. 

The link to some examples of that is here.

- [https://github.com/Zondax/wallet-core/blob/61b6e4a9733823fb0c04e048151df8febbcec4cf/trezor-crypto/src/bip32.c#L533](url)

- [https://github.com/Zondax/wallet-core/blob/61b6e4a9733823fb0c04e048151df8febbcec4cf/trezor-crypto/src/bip32.c#L806](url)